gsa/src/web/pages/alerts/component.js

```diff
@@ -891,6 +1005,20 @@ class AlertComponent extends React.Component {
                 onSave={this.handleCreateCredential}
               />
             }
+            {contentComposerDialogVisible &&
+              <ContentComposerDialog
+                includeNotes={parseInt(includeNotes)}
+                includeOverrides={parseInt(includeOverrides)}
```

Comment:
both values are boolean and contained in the state. So why not parsing them (parseYesNo I guess) when changing them in the state.

Reply:
The handler to change those values also handles non-boolean changes (filterId, filterString). Code changed to parse as YesNo instead of Int

Reply:
btw: parseYesNo does not return boolean but numbers.

Reply:
That's not completely correct. parseYesNo returns the boolean values for the protocol which are 1 and 0 as integers. We may decide to change them to true and false in GSA and in that case still using parseInt here would break it.